exes/mqtt-broadcaster/inc/config/mqtt_config.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+#include <variant>
+
+#include <glaze/glaze.hpp>
+#include <tfc/confman.hpp>
+
+namespace tfc::mqtt {
+
+enum struct port_e : std::uint16_t { mqtt = 1883, mqtts = 8883 };
+
+enum struct ssl { yes, no };
+
+// File under /etc/tfc/mqtt-broadcaster/def/mqtt_broadcaster.json which specifies the connection values for the MQTT broker.
```

Comment:
I am not exactly sure of the value of this comment here. Furthermore, it is duplicated in other files making it incorrect in those places.